teoriasegnali/fourier.tex

```diff
@@ -45,44 +44,46 @@ \section{Condizioni di esistenza Dirichlet}\index{serie!di Fourier!condizioni Di
 
 \section{Serie di Fourier di sinusoidi}
 Lo sviluppo in serie di Fourier può essere espresso come serie somma di seni e coseni o sinusoidi
-\[\begin{split} s(t)&=\sum_{k=0}^{+\infty}\left[ a_k \sen{2\pi\frac{k}{T}t} + b_k \cos{2\pi\frac{k}{T}t}\right] \\
-&=\sum_{k=0}^{+\infty}A_k\left[\sen{2\pi\frac{k}{T}t+\phi_k}\right] =\sum_{k=0}^{+\infty}A_k\left[\sen{2\pi\frac{k}{T}t}\Cos\phi_k +\cos{2\pi\frac{k}{T}t}\Sen\phi_k \right]
+\[\begin{split} s(t) &= \sum_{k=0}^{+\infty}\bigl( a_k \sen{2\pi kf_0 t} + b_k \cos{2\pi kf_0 t}\bigr) = \\
```

Comment:
non vedo la modifica a questa formula nella revisione
in tutto il documento vengono usate le parentesi \left( \right) e \left[ \right]  che si adattano automaticamente all'altezza del contenuto e consentono di scoprire parentesi non bilanciate a differenza delle varie versioni di \big, \Big, \bigg and \Bigg
https://en.wikibooks.org/wiki/LaTeX/Mathematics#Automatic_sizing